CHANGELOG.md

```diff
@@ -36,6 +36,15 @@ different versioning scheme, following the Haskell community's
   `MemoryStream.TryGetBuffer()` when cloning streams like is done when
   targeting .NET Standard 1.3+.
 
+### Bond compiler library ###
```

Comment:
I've intentionally moved the gbc section after the language projections: when people upgrade they care more about the changes for the projection languages and less about the Haskell library.